src/Genny.hpp

```diff
@@ -65,7 +65,10 @@ class Indent : public std::streambuf {
 class Object {
 public:
     Object() = delete;
-    explicit Object(std::string_view name) : m_name{name} {}
+    explicit Object(std::string_view name) : m_name{name} {
+        m_name.erase(std::remove(m_name.begin(), m_name.end(), '?'), m_name.end());
+        m_name.erase(std::remove(m_name.begin(), m_name.end(), '!'), m_name.end());
+    }
```

Comment:
Name sanitization is already implemented in newer versions of SdkGenny. See: https://github.com/cursey/sdkgenny/blob/bb400c07e6bb06dbac6029ae34f0a651d955f90c/include/Genny.hpp#L246

This project could be updated to the latest version of SdkGenny.